test/cypress/integration/test.acceptance.js

```diff
@@ -7,7 +7,7 @@ describe('Shepherd Acceptance Tests', () => {
   beforeEach(() => {
     Shepherd = null;
 
-    cy.visit('http://localhost:8080/test/dummy/', {
```

Comment:
I think we had to manually set this here, rather than using `baseURL` to make sure that things worked for both the demo app and the test app.

Reply:
I'm not sure I'm following. Do you mean running the demo app with `yarn start`? It doesn't seem like that would interfere with `cy`

Reply:
Updated

Reply:
I think it was to support the demo app, what we deploy for the demo.

Reply:
We can merge and see if the demo still looks good